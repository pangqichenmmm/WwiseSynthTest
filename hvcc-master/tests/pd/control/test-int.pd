#N canvas 36 23 399 349 10;
#X obj 50 53 loadbang;
#X msg 224 121 -1000.2;
#X obj 50 237 int 4.5;
#X msg 183 122 0;
#X msg 138 123 -34.3;
#X msg 50 124 56;
#X msg 88 123 2.4562;
#X msg 288 121 bang;
#X obj 50 75 t b b b b b b b;
#X obj 50 279 print;
#X connect 0 0 8 0;
#X connect 1 0 2 0;
#X connect 2 0 9 0;
#X connect 3 0 2 0;
#X connect 4 0 2 1;
#X connect 5 0 2 0;
#X connect 6 0 2 0;
#X connect 7 0 2 0;
#X connect 8 0 5 0;
#X connect 8 1 6 0;
#X connect 8 2 2 0;
#X connect 8 3 4 0;
#X connect 8 4 3 0;
#X connect 8 5 1 0;
#X connect 8 6 7 0;
