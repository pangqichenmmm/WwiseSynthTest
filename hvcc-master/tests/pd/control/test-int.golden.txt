[@ 0.000] print: 4
[@ 0.000] print: -1000
[@ 0.000] print: 0
[@ 0.000] print: -34
[@ 0.000] print: 2
[@ 0.000] print: 56
